nixos/modules/system/boot/initrd-ssh.nix

```diff
@@ -141,7 +150,7 @@ in
       }
 
       {
-        assertion = cfg.hostKeys != [];
+        assertion = (cfg.hostKeys != []) || cfg.ignoreEmptryHostKeys;
```

Comment:
`Empty` rather than `Emptry`

Reply:
Urgs, thanks! should not rename after testing :/